arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
@amarzavery @ravbhatnagar @athipp 
I pushed a new iteration which adds enums/x-ms-enum.
Noted that value set would be same at this moment to be consistent with response.
We would change value set to be different in future after ASPAAS team implement it.